pkg/util/kube/utils.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017, 2019 the Velero contributors.
+Copyright 2017, 2019, 2020 the Velero contributors.
```

Comment:
I think you can revert the change to this file now, since we're not going to sync VolumeSnapshots.